tests/configs/test-008-operator-restart-2.yaml

```diff
@@ -62,7 +62,7 @@ spec:
         spec:
           containers:
             - name: clickhouse-pod
-              image: yandex/clickhouse-server:21.1.7.1
+              image: altinity/clickhouse-server:21.8.altinity_prestable
```

Comment:
Remove all the references to altinity_prestable

Reply:
@alex-zaitsev 
The initial task was to test the prestable.
You mean yandex/clickhouse-server:21.8 ?